docs/core.adoc

```diff
@@ -0,0 +1,31 @@
+= AeroGear Services Core SDK
+
+The Core SDK provides helpers for parsing the configuration file and share common utilities between other SDK modules.
+
+It's main features are:
+
+* Parsing of the configuration file
+* Logging
+* Metrics gathering
+
+== Installation
+
+Use the Cordova CLI to add the Core module as a plugin to your project:
+
+----
+cordova plugin add @aerogearservices/Core
+----
+
+This will install the package from the NPM registry and properly link all native dependencies to the project's cordova Platforms.
+
+Refer to the link:./getting-started[Getting Started] guide for more information on installing the AeroGear SDK.
+
+== Usage
+
+After installation, the Core module will become available through the global `window.MobileCore` object in your project.
```

Comment:
So finally what should we use as global? I vote for `aerogear`.
`window.aerogear.metrics.MetricsService`
`window.aerogear.sync.login()`

Reply:
Yes, with what we're looking at from your work, I'll be changing this to `window.aerogear`